README.md

```diff
@@ -21,24 +32,48 @@ Currently the following variables are supported:
 
 ### General
 
-* `wipefs_become` - Default: true. If this role needs administrator
+* `disk_wipe_become` - Default: true. If this role needs administrator
   privileges, then use the Ansible become functionality (based off sudo).
-* `wipefs_become_user` - Default: root. If the role uses the become
+
+* `disk_wipe_become_user` - Default: root. If the role uses the become
   functionality for privilege escalation, then this is the name of the target
   user to change to.
 
+* `disk_wipe_disks` - Default: [] empty list.
```

Comment:
For clarification, this documentation should read that it expects a list of paths to disk block devices.

Reply:
updated